src/Uno.UI.Wasm/ts/WindowManager.ts

```diff
@@ -250,17 +264,46 @@
 			}
 
 			if (contentDefinition) {
-				for (const className of contentDefinition.classes) {
-					element.classList.add(`uno-${className}`);
+				let classes = element.classList.value; 
+				for (const className of uiElementRegistration.classNames) {
+					classes += " uno-" + className;
 				}
+
+				element.classList.value = classes;
 			}
 
 			// Add the html element to list of elements
 			this.allActiveElementsById[contentDefinition.id] = element;
 		}
 
+		public registerUIElement(typeName: string, isFrameworkElement: boolean, classNames: string[]): number {
+
+			const registrationId = Object.keys(this.uiElementRegistrations).length;
+
+			this.uiElementRegistrations[this.handleToString(registrationId)] = {
+				typeName: typeName,
+				isFrameworkElement: isFrameworkElement,
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The key 'isFrameworkElement' is not sorted alphabetically](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5247895)